src/NuGet.Core/NuGet.Configuration/PackageNamespaces/ConfigNameSpaceLookup.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace NuGet.Configuration
```

Comment:
Is result a better name?

Look-up suggestions a dictionary, where you'd need to ask it more questions to get what you need.

Reply:
How about `ConfigNameSpaceMatches`?

Reply:
This part removed.